### PR TITLE
operator: remove duplicate config creation

### PR DESCRIPTION
**Pull request by GrapeBaBa**
This commit remove duplicate invocation
k8sutil.InClusterConfig() in operator main

Signed-off-by: grapebaba <281165273@qq.com>

### CODE REVIEW

**Comment by xiang90** [comment]
@GrapeBaBa What is the problem with invoking `k8sutil.InClusterConfig()` twice?

File: pkg/util/k8sutil/k8sutil.go [hunk 1]
```diff
@@ -315,12 +315,7 @@ func InClusterConfig() (*rest.Config, error) {
 	return rest.InClusterConfig()
 }
 
-func NewTPRClient() (*rest.RESTClient, error) {
```
Comment (hongchaodeng) [inline pkg/util/k8sutil/k8sutil.go]: It's by design to make NewTPRClient() self-contained.
Again, what's the problem to call InClusterConfig() twice?

Comment (GrapeBaBa) [inline pkg/util/k8sutil/k8sutil.go]: oh, I didn't recognize this design. Not big issue, just looked the code and suspect it can be more elegant.

**Comment by GrapeBaBa** [comment]
Close this PR as it is not a issue.

**Pull request closed without merging**